src/commands/hueCommands.ts

```diff
@@ -9,15 +9,16 @@ export const HueSet: Command = {
   help: "hueSet 0000FF",
   description: "Changes Burn's hue light to a specific hex color",
   async execute(message: Message, args?: string[]) {
-    if(!args?.[1] || !RegExp('^[0-9A-F]{6}$', 'i').test(args[1])) {
-        message.channel.send('Please choose a hex color (no # please).')
+    const hex = args?.[1] && args[1].replace(/^#/, '');
```

Comment:
I'll double-check, but I believe optional chaining could actually swallow the other null check for `args?.[1]` here too:
```suggestion
    const hex = args?.[1]?.replace(/^#/, '');
```